src/app/file/tga_format.cpp

```diff
@@ -43,7 +43,8 @@ class TgaFormat : public FileFormat {
       FILE_SUPPORT_RGBA |
       FILE_SUPPORT_GRAY |
       FILE_SUPPORT_INDEXED |
-      FILE_SUPPORT_SEQUENCES;
+      FILE_SUPPORT_SEQUENCES |
+      FILE_SUPPORT_PALETTE_WITH_ALPHA;
```

Comment:
Check if this new addition doesn't add a new problem saving indexed files that contain palette entries with alpha (because TGA files doesn't support palettes with alpha, a least, that is what I remember). For example, the warning must be shown if the file is indexed and contain entries with alpha.